src/main/java/de/tum/cit/aet/artemis/buildagent/service/BuildJobContainerService.java

```diff
@@ -428,4 +442,17 @@ private Container getContainerForName(String containerName) {
         List<Container> containers = dockerClient.listContainersCmd().withShowAll(true).exec();
         return containers.stream().filter(container -> container.getNames()[0].equals("/" + containerName)).findFirst().orElse(null);
     }
+
+    private String getParentFolderPath(String path) {
```

Comment:
Can we use the actual Path class. The handling with slashes is hacky, since Windows uses \ instead of /

Reply:
This paths will be used inside a docker container so it'll be UNIX paths. At first I implemented using paths but it was unnecessarily long and complicated